pkg/cmd/render/render.go

```diff
@@ -148,6 +157,27 @@ func (r *renderOpts) Run() error {
 			return fmt.Errorf("unable to parse restricted CIDRs from config %q: %v", r.clusterConfigFile, err)
 		}
 	}
+	if len(renderConfig.ClusterCIDR) > 0 {
+		anyIPv4 := false
+		for _, cidr := range renderConfig.ClusterCIDR {
+			cidrBaseIP, _, err := net.ParseCIDR(cidr)
+			if err != nil {
+				return fmt.Errorf("invalid cluster CIDR %q: %v", cidr, err)
+			}
+			if cidrBaseIP.To4() != nil {
+				anyIPv4 = true
+				break
+			}
+		}
+		if !anyIPv4 {
+			// Single-stack IPv6 cluster, so listen on IPv6 not IPv4. IPv6
```

Comment:
Wouldn't it just be easier to put the quotes in the template? Extremely minor nit.

Reply:
ah, indeed. They're not required for IPv4 but of course they're not forbidden either